src/cli/cli.cpp

```diff
@@ -466,12 +483,24 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
      * Active
      * Done
      * @endcode
-     * @par api_copy
-     * #otBorderAgentIsActive
+     * @par
```

Comment:
Can you add another `@par` before this one to explain what the command does?  Since the api_copy was removed it just has the state list now.  Something like

Outputs the current status of the Border Agent service.

Reply:
Thanks. Fixed all in the new pushed commit